tests/login.test.js

```diff
@@ -0,0 +1,90 @@
+import SignIn from "../pages/SignIn"
+import HomePage from "../pages/HomePage"
+import SignUp from "../pages/SignUp";
+
+
+describe('Login', () => {
+    it('Deberia llevarte al SignIn', async ()=> {
+    await SignIn.abrir('bank')
+    await SignIn.username.setValue('federicodelucchi@gmail.com')
```

Comment:
You have a method in BasePage to perfom this action safely (the `clearValuesAndSendText()` one). Use that instead of directly calling an element method.